README.md

```diff
@@ -40,21 +40,22 @@ Therefore, this application will not run into any issues if it is restarted, res
+| AWS_REGION                   | Self-explanatory                                                                                                                                                                                                                                                              | no       | `us-west-2` |
+| ENVIRONMENT                  | If set to `dev`, will try to create the Kubernetes client using your local kubeconfig. Any other values will use the in-cluster configuration                                                                                                                                 | no       | `""`        |
+| EXECUTION_INTERVAL           | Duration to sleep between each execution in seconds                                                                                                                                                                                                                           | no       | `20`        |
+| EXECUTION_TIMEOUT            | Maximum execution duration before timing out in seconds                                                                                                                                                                                                                       | no       | `900`       |
+| POD_TERMINATION_GRACE_PERIOD | How long to wait for a pod to terminate in seconds; 0 means "delete immediately"; set to a negative value to use the pod's terminationGracePeriodSeconds.                                                                                                                     | no       | `-1`        |
+| METRICS_PORT                 | Port to bind metrics server to                                                                                                                                                                                                                                                | no       | `8080`      |
+| METRICS                      | Expose metrics in Prometheus format at `:${METRICS_PORT}/metrics`                                                                                                                                                                                                             | no       | `""`        | 
+| SLOW_MODE                    | If enabled, every time a node is terminated during an execution, the current execution will stop rather than continuing to the next ASG                                                                                                                                       | no       | `false`     |
+| CORDON_BEFORE                | If enabled, alle outdated nodes will get cordoned before any rolling update action. The default mode is to cordon a node just before draining it. See [#41](https://github.com/TwiN/aws-eks-asg-rolling-update-handler/issues/41) for possible consequences of enabling this. | no       | `false`     |
```

Comment:
Could you rename the environment variable to `EAGER_CORDONING`?

Reply:
Of course!